### ISSUE TITLE
Batch cancel order not encoding parameters properly

### ISSUE THREAD

**Issue by lrauldan** [binance/binance-futures-connector-python#34]
Though reported in #15 and closed as fixed, it is in fact still not encoding the query strings properly. I'm using 3.1.1 and the final query strings is still invalid:

```
[2022-06-02 08:44:58,565: DEBUG/ForkPoolWorker-8] https://testnet.binancefuture.com:443 "DELETE //fapi/v1/batchOrders?symbol=BTCUSDT&origClientOrderIdList=61cb41a6-a0b9-43d1-93d1-2a9050ae61cd&origClientOrderIdList=94e24acf-c3ba-4045-b359-1c9d3b7d75fd&origClientOrderIdList=dbd364ff-cf33-43a1-be48-1155c536e2f0&timestamp=1654159497710&signature=beaf2c54f6e3f23befdbc5661f97ea90d0992fa0b7f91c103d5c035f364e2eec HTTP/1.1" 400 84
```

**Comment by xiandong79** [comment]
@[andrea-c-binance](https://github.com/andrea-c-binance)

**Comment by mcsham** [comment]
> yes. I am facing the same issue
> 
> ```
> encoded_string: {'symbol': 'LRCUSDT', 'orderIdList': [14460401649, 14460401648], 'recvWindow': 2000, 'timestamp': 1657542266783}
> 
> query_string: symbol=LRCUSDT&orderIdList=14460401649&orderIdList=14460401648&recvWindow=2000&timestamp=1657542266783, special: False
> ```

client.cancel_batch_order(symbol=symbol, orderIdList=str(limit_lists), origClientOrderIdList=None)

))) but this bug still needs to be fixed.

**Comment by chairz-2** [comment]
Fixed in #51

**Comment by qgbcs** [comment]
@chairz-2  what is wrong?

```
fc.get_all_orders(symbol='ordi', limit=100)
---------------------------------------------------------------------------
ClientError                               Traceback (most recent call last)
Cell In[201], line 1
----> 1 fc.get_all_orders(symbol='ordi', limit=100)

File ~/anaconda3/lib/python3.11/site-packages/binance/cm_futures/account.py:477, in get_all_orders(self, **kwargs)
    474 url_path = "/dapi/v1/allOrders"
    475 params = {**kwargs}
--> 477 return self.sign_request("GET", url_path, params)

File ~/anaconda3/lib/python3.11/site-packages/binance/api.py:84, in API.sign_request(self, http_method, url_path, payload)
     82 query_string = self._prepare_params(payload)
     83 payload["signature"] = self._get_sign(query_string)
---> 84 return self.send_request(http_method, url_path, payload)

File ~/anaconda3/lib/python3.11/site-packages/binance/api.py:119, in API.send_request(self, http_method, url_path, payload)
    117 response = self._dispatch_request(http_method)(**params)
    118 self._logger.debug("raw response from server:" + response.text)
--> 119 self._handle_exception(response)
    121 try:
    122     data = response.json()

File ~/anaconda3/lib/python3.11/site-packages/binance/api.py:184, in API._handle_exception(self, response)
    182     if "data" in err:
    183         error_data = err["data"]
--> 184     raise ClientError(
    185         status_code, err["code"], err["msg"], response.headers, error_data
    186     )
    187 raise ServerError(status_code, response.text)

ClientError: (400, -1121, 'Invalid symbol.', {'Date': 'Sat, 27 Jan 2024 16:02:21 GMT', 'Content-Type': 'application/json', 'Content-Length': '38', 'Connection': 'keep-alive', 'Server': 'Tengine', 'X-MBX-USED-WEIGHT-1M': '40', 'x-response-time': '3ms', 'Access-Control-Allow-Origin': '*', 'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, OPTIONS'}, None)
```
